netmap/filter.go

```diff
@@ -270,3 +271,7 @@ func (f *Filter) MarshalJSON() ([]byte, error) {
 func (f *Filter) UnmarshalJSON(data []byte) error {
 	return (*netmap.Filter)(f).UnmarshalJSON(data)
 }
+
+func (f *Filter) id() string {
```

Comment:
Pls make it a function, it doesn't operate with private data and needed in very custom places.